src/main/java/seedu/address/model/Currency.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.model;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Represent currencies and its rate
+ */
+public class Currency {
+
+    private List<String> currencyCodes = Arrays.asList("SGD",
+            "AUD", "CAD", "CHF", "CNY", "GBP",
+            "HKD",  "ILS", "INR", "JPY", "MYR",
+            "NZD", "PHP", "SEK", "THB", "TWD",
+            "USD");
+
+    /**
+     * Check the currency code given contains in the list of currency codes
+     * @param currencyCode
+     * @return true is a given currency code is found in the list
+     */
+    public boolean containsCurrencyCode(String currencyCode) {
+        boolean isCurrencyCode = currencyCodes.stream().anyMatch(code -> code.equals(currencyCode));
+
+        return isCurrencyCode;
+    }
+
+    /**
+     * Retrieve the rate of the currency given the currency code
+     * The default based rate is in SGD (Singapore Dollar)
+     * @param currencyCode
+     * @return rate of the currency code
+     */
+    public double getCurrencyRate(String currencyCode) {
```

Comment:
Something you might want to consider about the currency rate in general: Given the context of your target audience (businessmen), would the accuracy of the currency rate be important? This is considering that the rates are defined here and changed only by the developer.